src/screens/expandedAssetSheet/components/sections/AboutSection.tsx

```diff
@@ -115,7 +115,7 @@ function Description({ text }: { text: string }) {
   const { accentColors } = useExpandedAssetSheetContext();
 
   const truncatedText = useMemo(() => truncate(text), [text]);
-  const canExpand = text.length > truncatedText.length;
+  const canExpand = text?.length > truncatedText?.length;
```

Comment:
Adding to comment above, The `Description` component is wrapped in a conditional render that checks if the text is falsy, so if the optional chaining is because of some sentry error there might be a different issue.